lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -47,6 +48,32 @@ function extractContentTypeAndValue(content: Content): [DialogType, string] {
   }
 }
 
+type ContentSelectorButtonProps = ComponentProps<typeof ButtonBase> & {
+  contentType?: string;
+};
+
+function ContentSelectorButton({
+  children,
+  contentType = 'pdf',
+  ...rest
+}: ContentSelectorButtonProps) {
+  return (
+    <ButtonBase
+      classes="w-full bg-stone-50 rounded-sm border border-stone-300 gap-x-2 items-center"
+      {...rest}
+    >
+      <div className="grow text-start p-2">
+        <strong className="text-slate-600">{children}</strong>
+      </div>
+      <div className="text-end p-2">
+        <span className="uppercase text-[11px] text-stone-500">
+          {contentType}
+        </span>
+      </div>
+    </ButtonBase>
+  );
+}
```

Comment:
Would it make sense to extract this component? On the one hand, this file is pretty big already, but on the other hand, it feels like a component too scoped to the content selector.